src/gsi/gsimod.F90

```diff
@@ -908,7 +908,7 @@ module gsimod
        
 ! 1/237: Dropsonde observations.
 
-! 292: SFMR observations.
+! 213: SFMR observations.
```

Comment:
292 is still referenced in gsimod.F90.  For example, namelist obsqc includes uv_doe_a_292 and uv_doe_b_292.   Shouldn't these now be uv_doe_a_213 and uv_doe_b_213?

Reply:
Updated.